zio-http/src/main/scala/zhttp/service/ServerResponseHandler.scala

```diff
@@ -126,3 +125,12 @@ private[zhttp] trait ServerResponseHandler[R] {
     } yield ()
   }
 }
+object ServerResponseHandler                  {
```

Comment:
```suggestion
object ServerResponseWriter                  {
```

We should rename the file to `ServerResponseWriter`